assets/app/views/_templates.html

```diff
@@ -0,0 +1,5 @@
+<h3>{{ title }}</h3>
+<ul class="templates" ng-repeat="template in list">
+  <template></template>
+</ul>
+<a href="">All {{ title }}</a>
```

Comment:
add a TODO for the href linking somewhere so that this doesn't get forgotten about